pythonSynapseUtils/auditAndUpdateByCommonDict.py

```diff
@@ -143,7 +142,7 @@ def updateAnnoByIdDictFromDict(syn,idDict,annoDict):
     
     """
     Update annotations from dictionary
-    by a given dict(key:annotation keys need to updated, value:a list of Synpase IDs)
+    by a given dict(key:annotation keys need to updated, value:a list of Synpase Objects)
```

Comment:
For readability, consider adding another function argument -- a flag for whether input is list of syn entity objects or not. Could set default = True.